multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt

```diff
@@ -20,7 +20,7 @@ class WPMangaReaderGenerator : ThemeSourceGenerator {
         SingleLang("Gabut Scans", "https://gabutscans.com", "id"),
         SingleLang("Martial Manga", "https://martialmanga.com/", "es"),
         SingleLang("Ngomik", "https://ngomik.net", "id", overrideVersionCode = 1),
-        SingleLang("Sekaikomik", "https://www.sekaikomik.xyz", "id", isNsfw = true, overrideVersionCode = 6),
+        SingleLang("Sekaikomik", "https://www.sekaikomik.in", "id", isNsfw = true, overrideVersionCode = 6),
```

Comment:
Need to bump the code too.

```suggestion
        SingleLang("Sekaikomik", "https://www.sekaikomik.in", "id", isNsfw = true, overrideVersionCode = 7),
```